waspc/cli/src/Wasp/Cli/Command/News/Listing.hs

```diff
@@ -0,0 +1,123 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Wasp.Cli.Command.News.Listing
+  ( -- * News Listing
+
+    -- This modules captures all decisions Wasp has to make when showing Wasp
+    -- news under different contexts (i.e., who requested the news listing):
+    --   - Which news to show under which context.
+    --   - Which news to mark as seen.
+    --   - Whether the user should interactively confirm they've read the news.
+    --   - etc.
+    --
+    -- We tried to capture most of the logic in pure function to simplify
```

Comment:
Hah I don't mind this comment, but we are in pure functional language, obviously we will try to do this always when we can, for multiple reasons, only one of them being easier testing :). So this comment is redundant.

Reply:
I added it because of this: https://github.com/wasp-lang/wasp/pull/3537#discussion_r2641343364

But am open to suggestions. 

Oh, you mean just the last part of the comment. Should I remove it?